### PR TITLE
add OpenTSDB to storage backends list

### PR DESCRIPTION
**Pull request by jakon89**

### CODE REVIEW

**Comment by DirectXMan12** [comment]
Why don't we just change this list to point to the `sink-owners.md` file or something of the sort?

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
Why don't we just change this entire paragraph to point to the `sink-owners.md` doc (or something similar) instead?

**Comment by jakon89** [comment]
@DirectXMan12 
not sure what you meant but PTAL. Thanks

File: README.md [hunk 1]
```diff
@@ -15,10 +15,9 @@ Heapster collects and interprets various signals like compute resource usage, li
 Heapster supports multiple sources of data.
 More information [here](docs/source-configuration.md).
 
-Heapster supports a pluggable storage backend.
-It supports [InfluxDB](http://influxdb.com) with [Grafana](http://grafana.org/docs/features/influxdb), [Google Cloud Monitoring](https://cloud.google.com/monitoring/), [ElasticSearch](https://www.elastic.co/products/elasticsearch), [Google Cloud Logging](https://cloud.google.com/logging/), [Hawkular](http://www.hawkular.org), [Riemann](http://riemann.io) and [Kafka](http://kafka.apache.org/).
+Heapster supports a pluggable storage backends described [here](docs/sink-owners.md).
```
Comment (DirectXMan12) [inline README.md]: `Heapster supports the pluggable...` (s/ a / the /).

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
one small nit, otherwise LGTM :+1:

**Comment by jakon89** [comment]
Updated. Thanks!

**Review by DirectXMan12 — APPROVED** [review]

**Comment by DirectXMan12** [comment]
I'd like to get an ok from @piosz as well (since this is a "front page" change, so to speak), but it looks good to me.

**Pull request merged**